src/.env-example

```diff
@@ -1,2 +1,3 @@
 DJANGO_SECRET_KEY=""
 DJANGO_SECRET_LOGIN_KEY=""
+GETGOV_PUBLIC_SITE_URL="https://federalist-877ab29f-16f6-4f12-961c-96cf064cf070.sites.pages.cloud.gov/site/cisagov/getgov-home/"
```

Comment:
Do we need this in a future `.env`? Or do we just need it in the docker-compose?

Reply:
I guess some clarity on docker-compose vs. manifest vs. .env would be good.